client/app/vps/additional-disk/index.js

```diff
@@ -0,0 +1,23 @@
+import component from './vps-additional-disk.component';
+
+import vpsAdditionnalDiskOrderModule from './order';
+
+const moduleName = 'vpsAdditionnalDiskModule';
```

Comment:
no need to suffix by `Module` but you might need to prefix by `ovhManager`
```suggestion
const moduleName = 'ovhManagerVpsAdditionnalDisk';
```